api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1407,22 +1407,23 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
```

Comment:
as i have already explained in previous comments the original data was failing because the class_id of the backing concept was not set to class frequency. The new data covers for that.